From: SJ Park <sj@kernel.org>
Date: Wed, 12 Aug 2026 06:35:36 -0700
Subject: [PATCH] ==== [PATCH v3 0/3] docs/mm/damon/design: add explanation of
 nr_snapshots ====

docs/mm/damon/design: add explanation of nr_snapshots

Add an explanation of nr_snapshots to avoid misunderstandings.

Changes from v3
- v3: https://lore.kernel.org/20260831150227.83416-1-sj@kernel.org
- None.
Changes from v2
- v2: https://lore.kernel.org/20260812104002.15225-1-aethernet65535@gmail.com
- Collect R-b: from SJ.
- Rebase to latest mm-new.
Changes from v1:
- v1: https://lore.kernel.org/damon/20260811101841.804596-1-aethernet65535@gmail.com
- Patch 1 word order adjustment: Repositioned "completely" ("tried to be
  completely applied" -> "completely tried to be applied").
- Patch 2: Use two spaces between sentences.
- Patch 3: Add the "Reviewed-by:" of SJ, because he added this in v1.
- Patch 3: Make some minor modifications, Patch 3 can be applied after
  Patch 2.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_ebb6dlk8 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_ebb6dlk8

diff --git a/hkml_cv_bogus/hkml_cv_bogus_ebb6dlk8 b/hkml_cv_bogus/hkml_cv_bogus_ebb6dlk8
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

